StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/DebugMessageHandler.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DebugMessage.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/2/24.
+//
+
+import Foundation
+
+// Emitted when the SDK should print to the console in debug mode.
+class DebugMessageHandler: ScriptMessageHandler<String> {
+    init(didReceiveMessage: @escaping (String) -> Void) {
```

Comment:
optional: What do you think about setting the default handler to `debugPrint`? This is generally the expected behavior, we just want to be able to mock it in tests.
```suggestion
    init(didReceiveMessage: @escaping (String) -> Void = Swift.debugPrint) {
```